pages/index.tsx

```diff
@@ -12,11 +12,14 @@ export default createPage({
     return (
       <>
         <Head>
-          <title>Digital Testimony Platform</title>
+          <title>Massachusetts Archive of Transparent Testimony</title>
         </Head>
```

Comment:
Oops, my bad! forgot to remove these head and title elements, now they're set in V2Layout. To update the titles and nav, we should remove the elements here and change the text in these two places:

https://github.com/codeforboston/advocacy-maps/blob/210acda4a66df206533fa17e42e698a81f1f95f6/components/V2Layout.tsx#L13
https://github.com/codeforboston/advocacy-maps/blob/210acda4a66df206533fa17e42e698a81f1f95f6/components/V2Layout.tsx#L28

Reply:
Done